lib/private/Metadata/Provider/ExifProvider.php

```diff
@@ -68,7 +68,12 @@ public function execute(File $file): array {
 		$size->setArrayAsValue([]);
 
 		if (!$data) {
-			$sizeResult = getimagesizefromstring($file->getContent());
+			try {
+				$sizeResult = getimagesizefromstring($file->getContent());
+			} catch (\Exception $ex) {
```

Comment:
```suggestion
			} catch (\TypeError $ex) {
```

As per https://github.com/php/php-src/commit/b9171d8a9354f8e86c43fe319f1c186c6590b2df